src/migrations/versions/0ebf5dad5713_.py

```diff
@@ -0,0 +1,42 @@
+"""empty message
+
+Revision ID: 0ebf5dad5713
+Revises: c5889224cafc
+Create Date: 2018-04-19 14:53:38.789190
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '0ebf5dad5713'
+down_revision = 'c5889224cafc'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('host', sa.Column('os_offer', sa.String(length=50), nullable=False))
+    op.add_column('host', sa.Column('os_sku', sa.String(length=50), nullable=False))
+    op.add_column('host', sa.Column('os_version', sa.String(length=50), nullable=False))
+    op.add_column('host', sa.Column('vm_type', sa.String(length=50), nullable=False))
+    op.create_index(op.f('ix_host_os_offer'), 'host', ['os_offer'], unique=True)
+    op.create_index(op.f('ix_host_os_sku'), 'host', ['os_sku'], unique=True)
+    op.create_index(op.f('ix_host_os_version'), 'host', ['os_version'], unique=True)
+    op.create_index(op.f('ix_host_vm_type'), 'host', ['vm_type'], unique=True)
```

Comment:
And yes, the initial problems I found with this were that I had existent records in the host table that had a non-nullable column being null because it was just added.

Reply:
Also unique shouldn't be true there! I'm not sure what this is.